src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -630,7 +630,7 @@ std::vector<std::string> disabledTestPatterns() {
         // TODO: Issue 92895
         // on platforms which do not support AMX, we are disabling I8 input tests
         retVector.emplace_back(R"(smoke_LPT/FakeQuantizeWithNotOptimalTransformation.CompareWithRefImpl.*CPU.*i8.*)");
-    if (!ov::with_cpu_x86_avx512_core_amx_bf16() && !ov::with_cpu_x86_bfloat16()) {
+    if (!ov::with_cpu_x86_avx512_core_amx_bf16() && !ov::with_cpu_x86_bfloat16() && !CPUTestUtils::with_cpu_x86_avx2_vnni_2()) {
```

Comment:
Probably these checks also need to be united under the umbrella of `is_<typename>_supported()` function to make it more readable. As for now, the condition is quite complicated and from the first glance it is not clear what does it actually mean.

Reply:
As far as I understand, I cannot reuse my new implemented helpers here because `ov_cpu_func_tests`  don't have access to `oneapi` (`mayiuse`). So I have to use here common helpers `ov::with_cpu_x86_<>()`